password-update-service-samples/README.md

```diff
@@ -0,0 +1,21 @@
+# Password Update Service Samples
+
+This repository contains sample implementations of password update services for [Asgardeo](https://console.asgardeo.io/) and [WSO2 Identity Server](https://wso2.com/identity-server/). These samples demonstrate how to create and configure a password update mechanism as an external service extension to the product.
+
+## Table of Contents
+- [Overview](#overview)
+- [Samples](#samples)
+- [Getting Started](#getting-started)
+
+## Overview
+The password update service samples in this repository are designed to help developers understand how to extend the password validation capabilities of Asgardeo and WSO2 Identity Server. Each sample provides an integration with [HaveIBeenPwned (HIBP)](https://haveibeenpwned.com/) and can be used as a reference for building your own custom password update services. The password update service can be used to enhance security by preventing users from using compromised passwords.
+
+## Samples
+### Express-based HIBP Service
+Located in [hibp-integration-express](hibp-intergration-express/README.md), this sample demonstrates how to implement a password update service that checks if the new password has been compromised in data breaches using the HaveIBeenPwned API using Node.js and Express.
```

Comment:
shall we term the directory as hibp-password-validator-service-express

Reply:
Fixed with 0a6405027d81ee3b63b9bc16b80ededc1973d230